src/etc/roles/literal/definitionRole.js

```diff
@@ -12,6 +12,12 @@ const definitionRole: ARIARoleDefinition = {
   prohibitedProps: [],
   props: {},
   relatedConcepts: [
+    {
+      concept: {
+        name: 'dfn',
```

Comment:
[`dfn` does not map to `definition`](https://www.w3.org/TR/html-aam-1.0/#details-id-29).